addon/components/age-bracket-chooser/template.hbs

```diff
@@ -1,4 +1,5 @@
 <label>{{label}}</label>
 {{item-chooser
   items=ageBracketValues multiple=multiple placeholder=placeholder
-  selection=_selection class=(if dark "item-chooser--dark") size=size}}
+  selection=_selection class=(if dark "item-chooser--dark") size=size
+  onSelect=onAgeBracketChange}}
```

Comment:
You didn't miss the `onAgeBracketChange` action ?

Reply:
a ok I see. Maybe add an empty `onAgeBracketChange`  in the component ?

Reply:
@tgallice it's passed by the parent. but i can at least declare it and raise an error if not passed